presto-iceberg/src/test/java/com/facebook/presto/iceberg/procedure/TestSetTablePropertyProcedure.java

```diff
@@ -160,6 +160,7 @@ public void testInvalidSetTablePropertyProcedureCases()
 
     private Table loadTable(String tableName)
     {
+        tableName = normalizeIdentifier(tableName, ICEBERG_CATALOG);
```

Comment:
Could all of these similar lines be in a base class?

Reply:
You mean loadTable() method?

Reply:
You could add a method to the base class like `loadTableAndNormalizeName` that just calls `normalizeIdentifier` then `loadTable` in the subclass. You would just have to update the existing calls to `loadTable`.